components/ErrorBanner/ErrorBanner.tsx

```diff
@@ -12,27 +13,52 @@ import {
 } from "./styles";
 
 export function ErrorBanner({ errorOrigin }: { errorOrigin?: ErrorOriginT }) {
+  const { isWrongChain } = useWalletContext();
   const { errorMessages, removeErrorMessage } = useErrorContext(errorOrigin);
 
-  if (!errorMessages.length) return null;
+  if (!errorMessages.length && !isWrongChain) return null;
 
   return (
     <Wrapper>
-      {errorMessages.map((message) => (
-        <ErrorMessageWrapper key={message?.toString()}>
+      {isWrongChain ? (
```

Comment:
I'm treating being on the wrong chain as a special case here, since there's not much point saying "something went wrong fetching the data" when they're on the wrong chain and everything will obviously fail